lbrynet/daemon/Downloader.py

```diff
@@ -99,12 +97,15 @@ def set_status(self, status, name):
 
     @defer.inlineCallbacks
     def check_fee_and_convert(self, fee):
-        max_key_fee_amount = self.convert_max_fee()
+        currency, amount = self.max_key_fee['currency'], self.max_key_fee['amount']
+        max_key_fee_amount = self.convert_max_fee(currency, amount)
         converted_fee_amount = self.exchange_rate_manager.convert_currency(fee.currency, "LBC",
                                                                            fee.amount)
```

Comment:
Doesn't the above two lines do the same thing but with different values? Why is `self.convert_max_fee()` called for one but `converted_fee_amount` calculated here? Can we drop `self.convert_max_fee` if it's not needed?

Reply:
>Doesn't the above two lines do the same thing but with different values?

By "above two lines", are you referring to the newly inserted lines, or are you referrring to the use of `self.convert_max_fee` and what `converted_fee_amount` is doing?

>Why is self.convert_max_fee() called for one but converted_fee_amount calculated here? 

Are you asking why `self.convert_max_fee` is not being used for `converted_fee_amount`? If so, I agree. I originally was not paying attention to that in order to keep as close to the way it was just with my changes included. 

That probably was not the best way to go about it. I can change it accordingly. 

>Can we drop self.convert_max_fee if it's not needed?

I would not say it is not needed. It can be used for both `max_key_fee_amount`  and `converted_fee_amount`.